docs/intro/tutorial01.txt

```diff
@@ -257,10 +257,11 @@ and put the following Python code in it:
     def index(request):
         return HttpResponse("Hello, world. You're at the polls index.")
 
-This is the simplest view possible in Django. To call the view, we need to map
-it to a URL - and for this we need a URLconf.
+To access this view in the browser, we need to map it to a URL - and for this
+we need what we call a URL configuration, or a "URLconf" for short.
 
-To create a URLconf in the polls directory, create a file called ``urls.py``.
+By convention, our URL configurations live inside a ``urls.py`` file. 
+The next step is to create this file inside the ``polls/`` directory.
```

Comment:
Since this is a tutorial, we don't need to provide as much detail to the readers. So I propose:
```suggestion
URL configurations are defined inside each Django app, and they are Python files named ``urls.py``.
```

Then, I think that we need to swap the next two blocks and show the content of the `urls.py` file immediately after the sentence above, and only after that, show the resulting folder hierarchy.

Reply:
I was skeptical about the swap at first, but in the end I think it's a good way to reassure the user 👍